pages/index.vue

```diff
@@ -1,26 +1,258 @@
 <template>
-  <section class="container">
-    <div>
-      <logo />
-      <h1 class="title">
-        akira
-      </h1>
-      <h2 class="subtitle">
-        The Linux Design Tool
-      </h2>
-      <div class="links">
-        <a href="https://nuxtjs.org/" target="_blank" class="button--green"
-          >Documentation</a
-        >
-        <a
-          href="https://github.com/nuxt/nuxt.js"
-          target="_blank"
-          class="button--grey"
-          >GitHub</a
-        >
+  <div class="wrapper">
+    <header>
+      <div>
+        <logo />
+        <nav>
+          <ul>
+            <li><a href="features">Features</a></li>
```

Comment:
these are not dedicated pages for now, let's add an anchor link to the relative sections in the page